docs/user-guide/src/capm3/pivoting.md

```diff
@@ -1,24 +1,21 @@
-# Move
+# CAPM3 Pivoting
 
-In the context of Metal3 Provider, _Move_ or _Pivoting_ is the process of moving
-CAPI and CAPM3 objects from one management k8s cluster to another. In Metal3,
-this process is performed using the
-[clusterctl](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html) tool
-provided by CAPI. For the rest of the document we will refer this process as
-_Move_.
+## What is pivoting
 
-The following objects are moved during this process:
+Cluster-api-providfer-metal3 (CAPM3) implements support for CAPI's 'move/pivoting' feature.
 
-**Main Objects:** Cluster, Machine, MachineSet, MachineDeployment,
-KubeadmControlplane, Metal3Cluster, Metal3Machine, and BareMetalHost
+CAPI Pivoting feature is a process for moving the provider components and declared Cluster API resources from a source
+management cluster to a target management cluster by using the clusterctl functionality called "move".
+More information about the general CAPI "move" functionality can be find [here](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html).
 
-**Related Objects:** Cluster Secrets, BareMetalHost Secrets, Configmaps,
-KubeadmConfig, Metal3MachineTemplate, Metal3IPPool, Metal3IPClaim, IPAddress,
-Metal3data, Metal3DataClaim, Metal3DataTemplate
+In Metal3, pivoting is performed by using the CAPI clusterctl tool provided by Cluster-API. clusterctl recognizes pivoting as move.
+During the pivot process clusterctl pauses any reconciliation of Cluster-API objects and this gets propagated to (CAPM3) objects as well.
+Once all the objects are paused, the objects are created on the other side on the target cluster and deleted from the
+ephemeral cluster.
 
-## Prerequisite
+## Prerequisits
 
-* **`clusterctl` is used to initialize both the bootstrap and target cluster.**
+* **It is mandatory to use `clusterctl` both the bootstrap and target cluster.**
 
   If the provider components are not installed using `clusterctl`, it will not
   be able to identify the objects to move.  Initializing the cluster using
```

Comment:
Since it is hard to show what changes are requested, I will mark them in both.

So if the clusters are not initialized using clusterctl, all the CRDS of the objects to be moved to target cluster need to have these labels both **before performing move**. Note: **this** is not recommended, since the way clusterctl identifies objects to manage might change in future, so it’s always safe to install CRDs and controllers through **`clusterctl init`**.

Reply:
done